test_windows.js

```diff
@@ -0,0 +1,19 @@
+const spawn = require('child_process').spawn;
+var path = require('path');
+var semver = require('semver');
+
+process.env.GCLOUD_PROJECT = 0
+process.env.CLOUD_DEBUG_ASSERTIONS = 1
+spawn(path.join('node_modules', '.bin', 'mocha.cmd'),
+  [ path.join('build', 'test'), '--timeout 4000', '--R' ], {
+  stdio : 'inherit'
+}).on('close', function() {
```

Comment:
I think listening for [exit](https://nodejs.org/api/child_process.html#child_process_event_exit) might be better here.  The function would have parameters for the exit code and signal.  For each `spawn`, we need to record the exit code and exit from the `test_windows.js` script with a non-zero exit code if either `spawn` fails so that Appveyor records the test as a failure.